crates/dbs-virtio-devices/Cargo.toml

```diff
@@ -32,7 +32,7 @@ serde_json = "1.0.9"
 thiserror = "1"
 threadpool = "1"
 virtio-bindings = "0.1.0"
-virtio-queue = "0.4.0"
+virtio-queue = "0.6.0"
 vmm-sys-util = "0.11.0"
 vm-memory = { version = "0.9.0", features = [ "backend-mmap" ] }
```

Comment:
Does this need to be updated too?

Reply:
I think there is no need. Since `virtio-queue` 0.6.0 use `vm-memory` 0.9.0 and 0.10.0 of vm-memory may cause conflict.